test/required_files/agent-deb.txt

```diff
@@ -0,0 +1,14 @@
+/lib/systemd/system/datadog-agent-security.service
+/lib/systemd/system/datadog-agent-process.service
+/lib/systemd/system/datadog-agent.service
+/lib/systemd/system/datadog-agent-trace.service
+/lib/systemd/system/datadog-agent-sysprobe.service
+/etc/init.d/datadog-agent-process
+/etc/init.d/datadog-agent-security
+/etc/init.d/datadog-agent
+/etc/init.d/datadog-agent-trace
+/etc/init/datadog-agent.conf
+/etc/init/datadog-agent-sysprobe.conf
+/etc/init/datadog-agent-trace.conf
+/etc/init/datadog-agent-process.conf
+/etc/init/datadog-agent-security.conf
```

Comment:
I think these 5 upstart files are also present in the rpm packages, as they are added [here](https://github.com/DataDog/datadog-agent/blob/5b9a9a8389e3afbc8362a8e56852f0a0a7349070/omnibus/config/software/init-scripts-agent.rb#L58-L82).

It's debatable whether we still need them (since they were here for RHEL 6), but until we decide to remove them, I'd say we should still verify their presence?

Reply:
Indeed they should still be checked for. I added those to the required list